src/graphql/mutation/index.ts

```diff
@@ -0,0 +1,16 @@
+
+import { gql } from "@apollo/client";
+
+//Login do Usuário Mutation
+export const LOGIN_MUTATION = gql`
+  mutation Login($email: String!, $password: String!) {
+    login(data: { email: $email, password: $password }) {
```

Comment:
nit: poderia manter o tipo criado no graphql
```suggestion
  mutation Login($data: LoginInput!) {
    login(data: $data) {
```

Reply:
Irei alterar.

Reply:
Alteração realizada